### ISSUE TITLE
Tool zur Entfernungsmessung

### ISSUE THREAD

**Issue by oliver** [ffansbach/de-map#17]
Es wäre praktisch, wenn die Karte einen Entfernungsmesser integriert hätte, um Entfernungen zwischen potentiellen Freifunk-Standorten ausmessen zu können. Sowas ist manchmal nötig, und gute Seiten dafür sind im Netz erstaunlich schwierig zu finden. Eine Integration in die Freifunk-Karte wäre genau der richtige Platz dafür.

Hab auf die Schnelle folgende Leaflet-Plugins dafür gefunden:
* https://github.com/jtreml/leaflet.measure (Demo: http://jtreml.github.io/leaflet.measure/example.html)
* https://github.com/makinacorpus/Leaflet.MeasureControl (Demo: http://makinacorpus.github.io/Leaflet.MeasureControl/)
* https://github.com/shramov/leaflet-plugins
* https://github.com/ljagis/leaflet-measure (Demo: http://ljagis.github.io/leaflet-measure/)

**Comment by viisauksena** [comment]
hey @oliver . hast du es geschafft ein leaflet so einzubauen das das klappt?

**Comment by oliver** [comment]
Nein, ich hab das gar nicht ausprobiert - hab mit dem ganzen Thema nichts weiter gemacht, und hab im Moment auch nicht vor, da aktuell was zu machen.
